easybuild/scripts/bootstrap_eb.py

```diff
@@ -519,6 +532,8 @@ def main():
 # EasyBuild is a (set of) Python packages, so it depends on Python
 # usually, we want to use the system Python, so no actual Python dependency is listed
 allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+preinstallopts = '%(preinstallopts)s'
```

Comment:
there are a couple of reasons why we can't (easily) use the readily available EasyBuild easyconfig files; this `preinstallopts` part, which is specific to the bootstrap, is one

only knowing the version to include in here *after* stage1 (which installs the latest EasyBuild packages (+ vsc-base) from PyPI), is another